iidm/iidm-modification/src/test/java/com/powsybl/iidm/modification/tripping/BranchTrippingTest.java

```diff
@@ -47,12 +50,14 @@ void lineTrippingTest() {
         assertFalse(line.getTerminal2().isConnected());
 
         LineTripping unknownLineTripping = new LineTripping("NOT_EXISTS");
-        Exception e1 = assertThrows(PowsyblException.class, () -> unknownLineTripping.apply(network));
+        Exception e1 = assertThrows(PowsyblException.class, () -> unknownLineTripping.apply(network, true, ReportNode.NO_OP));
         assertEquals("Line 'NOT_EXISTS' not found", e1.getMessage());
+        assertDoesNotThrow(() -> unknownLineTripping.apply(network));
```

Comment:
We should add a section in the PR description to list all the NetworkModifications that threw an Exception in all cases and that now only throw one when `throwException` is true.